src/containers/artistas/UpdateArtista.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { firebaseConnect, isLoaded, isEmpty } from 'react-redux-firebase';
+
+import ArtistaForm from '../../components/artistas/ArtistaForm.jsx';
+
+const UpdateArtista = ({ firebase, artista, id }) => {
+
+  if (!isLoaded(artista)) { // If not loaded…
+    return 'Loading'; // …show 'loading'
+  } else if (isEmpty(artista)) { // …else. If is empty…
+    return 'Error'; // …show 'Error?'
+  } else {
+    return (
+      <section>
+        <header className='grid-row margin-bottom-basic'>
+          <div className='grid-item'>
+            <h1 className='font-size-large'>Editar Entrada</h1>
+          </div>
+        </header>
+
+        <ArtistaForm id={id} artista={artista} />
+      </section>
+    );
+  }
+};
+
+export default compose(
+  withRouter,
```

Comment:
@varffs can u test if this is needed here? Remove it and try going to the artist list, then to edit an artist then hit the back button and the link in the nav. try having another tab open on this view and check if the data updates properly while editing on another tab.